doc/user/workflow.md

```diff
@@ -25,7 +28,7 @@ each job.
 Another alternative use case is to emulate a parallel file system by creating
 one big pool with a single POSIX container to be accessed by all users.
 
-## `daos` Utility
+## `daos(1)` Utility
```

Comment:
What does the (1) signify here? Is it supposed to be a reference?

Reply:
It is referring to the section of the man page to check.